site/_posts/2021-04-23-announcing-fusionauth-1-26.md

```diff
@@ -0,0 +1,103 @@
+
+## Advanced multi-factor authentication (MFA)
+
+This is a premium feature requiring a [paid license](/pricing/editions/). It increases support for MFA in FusionAuth, including adding the following features:
+
+* customizable SMS templates, including localization
+* a new MFA factor: email 
+* support for multiple SMS delivery mechanisms
+* available MFA options can be specified at the tenant level
+* multiple authentication methods can be associated with an account
+* one-time use recovery codes
+* self service MFA management
+
+It's important to note that the FusionAuth team wants every application user to have access to MFA. Therefore, TOTP/Google Authenticator based MFA remains available in the community edition. 
+
+Here's a customized SMS template:
+
+{% include _image.liquid src="/assets/img/blogs/release-1-26/custom-sms-template.png" alt="A customized SMS MFA template." class="img-fluid" figure=false %}
+
+Here's the tenant edit view, where I am configuring available MFA methods:
+
+{% include _image.liquid src="/assets/img/blogs/release-1-26/setting-mfa-options.png" alt="Configuring allowed MFA methods." class="img-fluid" figure=false %}
+
+Unfortunately, these changes are **not backwards compatible** with the previous Two Factor implementation, so if you have built on top of it, please [review the changes carefully](/docs/v1/tech/apis/two-factor/).
+
+## Entity Management
+
+When talking to customers, users and potential clients, we heard that they were often modelling entities beyond users and wanted to use FusionAuth. So, we built it; Entity Management is a premium feature requiring a paid license. It is currently in technology preview and may change; please provide feedback to us. 
+
+You can create distinct entity types representing classes of entities, such as "API" or "Company". You then can create permissions on these types, such as "invoke" or "manage". At that point, you create individual instantiations of these types, such as "Reminder API" or "Bachmanity". Finally, create "grant" relationships between entities, which allow you to model permissions between entities.
+
+For example, suppose I have a reminder API which needs to call a todo API to query upcoming tasks. With entity management, I can:
+
+* Create an "API" entity type with the following permissions: `query`, `execute`, and `configure`.
+* Create an "reminder API" entity
+* Set up a "todo API" entity
+* Grant the reminder API `query` permissions on the todo API
+
+When the reminder API is ready to gather upcoming tasks, it "signs in" with the Client Credentials grant. FusionAuth will, if the reminder API has the appropriate permission on the todo API, create a JWT. The reminder API can present the JWT, perhaps as a bearer token, to the todo API. The todo API can then process the request and return the needed data.
```

Comment:
I am a visual person when learning flows, so does this capture the flow you were envisioning?  Feel free to use it if you think it adds value 
https://www.figma.com/file/x0hZ3peCmTXkhDRKn1arng/entity-management?node-id=0%3A1


![flow-reminders-todo2](https://user-images.githubusercontent.com/16090626/116586147-6079e100-a8d6-11eb-97f7-34f907e53005.png)

Reply:
That's a great diagram. Very close, I'd just wrap the client cred grant in a 'FusionAuth' logo, whereas the two APIs are code running in the application (represented by Entities in FusionAuth the same way there are applications and FusionAuth applications).